README.md

```diff
@@ -172,13 +176,11 @@ as long as you have one of the 3 supported keywords right in front
 of the mandatory marker):
 
 ```
-// @todo #224
-/* @todo #TEST-13 */
```

Comment:
Why it was removed?

Reply:
It is not supported in the code, there is only a logic to remove prefix, but no logic to remove any kinds of symbols from the end

Reply:
In this case, I'd suggest to fix/improve support for such type of comments instead of removing it from documentation.

Reply:
There is a bug about it https://github.com/cqfn/pdd/issues/225